# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1272489577 14400
#      Wed Apr 28 17:19:37 2010 -0400
# Node ID c1111769f59570f183f9831ae631be4b2e7d697e
# Parent  eb69d94e8648be1f86e93bbf3e8aa065e4f46dca
test_struct: update expected error message text

diff --git a/test/ChangeLog b/test/ChangeLog
--- a/test/ChangeLog
+++ b/test/ChangeLog
@@ -1,8 +1,12 @@
+2010-04-28  John W. Eaton  <jwe@octave.org>
+
+	* test_struct.m: Update expected error message text.
+
 2010-03-19  Jaroslav Hajek  <highegg@gmail.com>
 
 	* test_slice.m: Use ID check rather than message for invalid resizing.
 
 2010-03-05  Jaroslav Hajek  <highegg@gmail.com>
 
 	* test_logical-wfi-f.m: Update.
 	* test_logical-wfi-t.m: Update.
diff --git a/test/test_struct.m b/test/test_struct.m
--- a/test/test_struct.m
+++ b/test/test_struct.m
@@ -175,17 +175,17 @@
 %!    deal (1, 2, 3, 4, "5", "6", "7");
 %!  assert ({b.name}, {1, 2, "b", "e", 3, 4});
 %!  assert ({b.value}, {100, "5", 100, "6", 100, "7"});
 
 %!error <a cs-list cannot be further indexed>
 %!  [a(1:3).x] = deal ([1, 5], [3, 7], [8, 9]);
 %!  [a(2:3).x(2)] = deal (10, 11);
 
-%!error <can't perform indexing operations for cs-list type>
+%!error <a cs-list cannot be further indexed>
 %!  [a(1:3).x] = deal ([1, 5], [3, 7], [8, 9]);
 %!  a(2:3).x(2);
 
 %!error id=Octave:index-out-of-bounds
 %!  a(1).x.x = 1;
 %!  a(2).x;
 
 %!error <invalid number of output arguments for constant expression>
